src/style.css

```diff
@@ -1,3 +1,62 @@
+:root {
+  --link-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo-img {
+  max-width: 40px;
+  max-height: 40px;
+}
+
+.nav-list {
+  padding: 0;
+  display: flex;
+  list-style: none;
+  margin: 0;
+}
+
+.nav-link {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  margin-left: 20px;
```

Comment:
still not resolved, move it to `nav-item` styles
![image](https://github.com/mate-academy/layout_moyo-header/assets/55200530/d35eb49b-6ca3-4c40-81e7-5e35d02ed569)